app/state/recurringReservations.spec.js

```diff
@@ -1,19 +1,14 @@
 import { expect } from 'chai';
 
-import recurringReservations from './recurringReservations';
+import recurringReservations, { initialState, populateReservations } from './recurringReservations';
 
 describe('state/recurringReservations', () => {
   describe('reducer', () => {
     const reducer = recurringReservations.reducer;
 
     it('returns correct initial state', () => {
-      const expected = {
-        baseTime: null,
-        frequency: '',
-        numberOfOccurrences: 1,
-      };
       const actual = reducer(undefined, { type: 'NOOP' });
-      expect(actual).to.deep.equal(expected);
+      expect(actual).to.deep.equal(initialState);
     });
 
```

Comment:
- [ ] I think initialState values should be test here literally. (without importing it)